src/views/bootablevolumes/list/BootableVolumesList.tsx

```diff
@@ -51,10 +65,23 @@ const BootableVolumesList: FC = () => {
     }));
   };
 
+  const createItems = {
+    form: t('With form'),
+    yaml: t('With YAML'),
+  };
+
+  const onCreate = (type: string) => {
+    return type === 'form'
+      ? createModal((props) => <AddBootableVolumeModal {...props} />)
+      : history.push(`/k8s/ns/${namespace || DEFAULT_NAMESPACE}/${DataSourceModelRef}/~new`);
+  };
+
   return (
     <>
-      <ListPageHeader title={t('Bootable volumes')} badge={<DeveloperPreviewLabel />}>
-        <AddBootableVolumeButton buttonVariant={ButtonVariant.primary} />
+      <ListPageHeader title={t('Bootable resources')} badge={<DeveloperPreviewLabel />}>
+        <ListPageCreateDropdown items={createItems} onClick={onCreate}>
```

Comment:
Please add the prop createAccessReview, should look like this:
createAccessReview={{ groupVersionKind: DataSourceModelRef, namespace: namespace }} - namespace is optional

Reply:
Yes, I thought about that. But on this page, we have DataSources and PVC. 
I can fetch if the user has create permission for DS OR PVC and in that case, show the button. 
In the modal i can show just PVC if the user has no create permissions to DS or vice-versa. 
What do you think? Maybe i can do that in a separate pr.